package.json

```diff
@@ -8,5 +8,8 @@
     "@babel/preset-env": "^7.1.0",
     "babel-plugin-add-module-exports": "^1.0.0",
     "lerna": "^3.4.3"
+  },
+  "scripts": {
+    "new-plugin": "./scripts/new-plugin.sh"
```

Comment:
running this locally shows a couple of weird behaviors (shown is two runs, with names `Kyle Plugin` and `Test Plugin`):
1. my plugin's name seems to be mangled at the first character 
2. a runaway file copy seems to be creating copies with trailing singlequotes

<img width="283" alt="image" src="https://user-images.githubusercontent.com/680248/107134354-f9d8be80-68b6-11eb-90ec-83aa41813e9b.png">

Reply:
Ok, looks like this was a bad sed line. Frustratingly, mac sed _requires_ you to supply a backup prefix. I tried to get around that by just supplying an empty string, which I think would normally work, but I did it in a weird way.

Anyway, I believe it's now fixed. Grab a look and see what you think.